.circleci/config.yml

```
@@ -178,6 +177,23 @@ jobs:
                 --build-arg VERSION="${CIRCLE_TAG:-$THISVERSION}" . \
               && e=0 && break || sleep 15
             done && [ "$e" -eq "0" ]
+      - run:
+          name: Check Docker image and bump wrapper version
+          no_output_timeout: 60m
+          command: |
+            export PY3=$(pyenv versions | grep '3\.' |
+                         sed -e 's/.* 3\./3./' -e 's/ .*//')
+            pyenv local $PY3
+            # Get version, update files.
+            THISVERSION=$( python3 get_version.py )
+            BUILT_VERSION=$( docker run --rm -it nipreps/fmriprep:latest --version )
```

interactive/tty option can clutter the output
```suggestion
            BUILT_VERSION=$( docker run --rm nipreps/fmriprep:latest --version )
```